src/Core/Feature.js

```diff
@@ -135,14 +135,13 @@ export class FeatureGeometry {
 
     /**
      * Push new coordinates in vertices buffer.
-     * @param {Coordinates} coordIn The coordinates to push.
      * @param {Feature} feature - the feature containing the geometry
+     * @param {Coordinates} coordIn The coordinates to push.
      */
-    pushCoordinates(coordIn, feature) {
+    pushCoordinates(feature, coordIn) {
```

Comment:
this change seems not necessary and it breaks the API, I would prefer that we dont do it.

Reply:
I did this change for the 2 methods: pushCoordinates() and pushCoordinatesValues() to be more similaire as both do the same thing, the second one had been added for the unique case of vectorTile support. The purpose at he end is to have only 1 methode to do both.

Reply:
It would be the topic of a new PR aiming at clearing all the process of creating new itowns.Features I think.
It's in the idea mostly link with the way how is handle base_altitude.

Reply:
I think it would be interresting to do at least what I propose in my previous comment in this PR so we don't break the API.

Reply:
Thanks! Can you do the same for `pushCoordinatesValues` please?

Reply:
Also, can you add `BREAKING CHANGE: description` in the commits that modify these methods please?

Reply:
1) I didn't change the way pushCoordinatesValues() is called, there is no deprecation there.
2) I did add the BREAKING CHANGE message but indeed not in the good commit. I will correct that mistake now.

Reply:
1. From what I see you moved its signature from `pushCoordinatesValues(feature, long, lat, normal)` to `pushCoordinatesValues(feature, coordIn, coordProj)`